content/blog/2020-03-16-teaching-online-on-short-notice/index.markdown

```diff
@@ -0,0 +1,229 @@
+    The format…encourages participants who want to contribute to say more and listen less."
+    The solution is to run a text chat beside the video conference
+    where people signal that they want to speak,
+    and then call on them in turn.
+
+1.  *Call on people by name.*
+    "Any questions?" often produces an awkward silence,
+    and even when it doesn't,
+    it gives fluent extroverts undue air time.
+    Instead,
+    call on people from your selected group by name.
+    (If your video conferencing tool doesn't display real names by default,
+    ask participants to give theirs when they ask questions in the chat.)
+    This also helps even out the effects of differing ability:
+    you can throw easy questions at people who are struggling
+    and save the hard ones for people who might otherwise get bored.
+
+1.  *Don't teach alone.*
+    Every great surgeon has a great nurse beside her.
+    Equally,
+    when you're teaching online for the first time,
+    get someone else to monitor the chat for questions,
+    figure out who to call on next,
+    and so on.
+    If none of your colleagues can do this (remember, they're probably scrambling too),
+    ask one of your more advanced learners to help out.
+    Whoever it is,
+    make sure to introduce them at the start of class.
+
+1.  *Do a dry run (with your copilot).*
+    Fifteen minutes with a small group before you do a real (larger) class
+    can save you from saying, "Nobody told me there was a firewall."
+
+1.  *Keep your notes beside you.*
+    For whatever reason,
+    people seem less distracted when they look to the side at a printed copy
+    than when they look up at a second monitor.
+    There's also less risk of your audience getting to see the inside of your nose…
+
+1.  *Add speaker's notes to your slides.*
+    These can be point-form or long-form depending on how much time you have,
+    but they will make your content easier to find,
+    easier to review,
+    and more accessible to people who have trouble seeing or hearing.
+
+1.  *Use accessibility tools to help everyone.*
+    Speaker's notes are just one example of this:
+    enlarging your cursor,
+    using a [key caster][keycastr] to display control keys as you type,
+    switching to high-contrast mode all help compensate for shaky video
+    or low-quality screens.
+
+1.  *Teach from your smallest screen.*
+    If your slides or your code are on the fancy meter-wide monitor you use for your daily work,
+    they're not going to look good when they're squashed down
+    to fit in the 800-pixel window your learners have on their tablets.
+    Using a smaller video window also reduces bandwidth a little,
+    which may in turn help learners in lower-quality environments.
+
+1.  *Your voice matters more than your face.*
+    You can switch off video if you have to,
+    but if you lose sound,
+    your class is over.
+    Eight dollars will get you a Yeti or Samson Meteor microphone;
```

Comment:
is this supposed to be eighty? Yeti are currently $130 on amazon

Reply:
Eight for a Samson - I'll amend.